CorsixTH/Src/persist_lua.cpp

```diff
@@ -555,7 +548,7 @@ class LuaPersistBasicWriter : public LuaPersistWriter
             {
                 if(lua_objlen(L, iIndex) != 0)
                 {
-                    setError("Can only persist non-empty userdata if "
+                    set_error("Can only persist non-empty userdata if "
                         "they have a __depersist_size metafield");
```

Comment:
Should such multi-line strings be merged onto a single line at some point?

Reply:
Why?